synfig-studio/src/gui/cellrenderer/cellrenderer_value.cpp

```diff
@@ -252,6 +252,17 @@ bool get_paragraph(synfig::String& text)
 	return App::dialog_paragraph(_("Text Paragraph"), _("Enter text here:"), text);
 }
 
+bool
+is_layer_deleted(const synfig::Layer::ConstHandle& layer)
+{
+	Canvas::LooseHandle layer_canvas = layer->get_canvas();
```

Comment:
Looks good, but I think this part could be clearer:
```
	if (Canvas::LooseHandle layer_canvas = layer->get_canvas())
		return std::find(layer_canvas->begin(), layer_canvas->end(), layer) == layer_canvas->end();

	return true;
```

Anyway you can leave it as-is.
Please rebase.

Reply:
Refactored and rebased.